python/graphstorm/model/edge_gnn.py

```diff
@@ -147,11 +152,14 @@ def edge_mini_batch_gnn_predict(model, loader, return_label=False):
     loader : GSgnnEdgeDataLoader
         The GraphStorm dataloader
     return_label : bool
-        Whether or not to return labels.
+        Whether or not to return labels
+    return_proba : bool
+        Return all the predicted results when true otherwise only the maximum value
 
     Returns
     -------
-    Tensor : GNN prediction results.
+    Tensor : GNN prediction results. Return all the results when return_proba is true
```

Comment:
Does it return two tensors?

Reply:
Yes, depending on the `return_label` and `return_proba` flag.